src/middlewares/upload.js

```diff
@@ -0,0 +1,32 @@
+const multer = require('multer');
+const util = require('util');
+
+const storage = multer.diskStorage({
+  destination: (requset, file, callback) => {
+    callback(null, './src/uploads/images');
```

Comment:
You could put a sample file/image here because git won't upload an empty directory.